src/components/Accordion/Accordion/Accordion.jsx

```diff
@@ -4,62 +4,77 @@ import cx from "classnames";
 import useMergeRefs from "../../../hooks/useMergeRefs";
 import "./Accordion.scss";
 
-const Accordion = forwardRef(({ children: originalChildren, allowMultiple, defaultIndex, className, id }, ref) => {
-  const componentRef = useRef(null);
-  const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+const COMPONENT_ID = "monday-accordion";
```

Comment:
consider using `ELEMENT_TYPES` along with `getTestId` (see [here](https://github.com/mondaycom/monday-ui-react-core/blob/master/src/utils/test-utils.js))

Reply:
I've opened some discussion about data-testid.
So for a while I will not perform changes, after we'll make decision I'll fix it in separate PR.